resources/assets/sass/components/_offer.scss

```diff
@@ -0,0 +1,77 @@
+// Offer
+//
+// Offer block
+//
+// Markup:
+//<div class="c-offer {$modifiers}">
+//    <h3 class="c-offer__title">Edasi-tagasi Riiast või Helsingist Bangkoki al 350 €</h3>
+//    <div class="c-offer__image-wrap">
+//        <img src="assets/img/offer.jpg" alt="" class="c-offer__image">
+//    </div>
+//</div>
+//
+// default   - An offer.
+//.m-small - Smaller offer
+//
+// Styleguide 5
+
+.c-offer {
+
+	@include background-color-dark-modifiers;
+
+	position: relative;
+	display: block;
+	height: 192px;
+	width: 100%;
+	background: $color-fill-neutral;
+
+	&.m-small {
+
+		height: 144px;
+	}
+}
+
+	.c-offer__title {
+
+		@include vertical-center;
+
+		color: $color-white;
+		font-size: $font-size-large;
+		font-weight: $font-weight-bold;
+		line-height: 1.15;
+		text-align: center;
+		position: absolute;
+		z-index: 2;
+		width: 100%;
+		padding: 0 48px;
+	}
+
+	.c-offer__image-wrap {
+
+		position: relative;
+		overflow: hidden;
+		height: 100%;
+
+		&:before {
+
+			content: '';
```

Comment:
Properties should be ordered background, content, height, left, position, top, width, z-index